djstripe/models/base.py

```diff
@@ -406,14 +407,23 @@ def _stripe_object_field_to_foreign_key(
         """
         field_data = None
         field_name = field.name
-        raw_field_data = manipulated_data.get(field_name)
+        raw_field_data = manipulated_data.get(field_name, "NA")
```

Comment:
Not like this please. This obviously breaks if the value is ever `"NA"`. Do a proper conditional variable.

`some_conditional = field_name not in manipulated data`